ui/src/components/erc/Inspect/Inspect.js

```diff
@@ -4,12 +4,12 @@ import { ReflexContainer, ReflexElement, ReflexSplitter } from 'react-reflex';
 import CodeView from './CodeView/CodeView';
 import DataView from './DataView/DataView';
 import SelectFile from './SelectFile/SelectFile';
-import config from '../../../helpers/config';
+//import config from '../../../helpers/config';
```

Comment:
If something is not needed anymore, please remove, or add a further comment why this might be needed in the future.